{{cookiecutter.project_slug}}/.neuro/live.yml

```diff
@@ -0,0 +1,158 @@
+kind: live
+title: {{ cookiecutter.project_name }}
+id: PROJECT_SLUG_WITH_UNDERSCORES_PLACEHOLDER
+
+defaults:
+  preset: gpu-small-p
+  life_span: 1d
+  tags:
+    - kind:flow
+    - flow-id-with-dashes:{{ cookiecutter.project_slug }}
+
+volumes:
+  data:
+    remote: storage:$[[ flow.flow_id ]]/data
+    mount: /project/data
+    local: data
+  code:
+    remote: storage:$[[ flow.flow_id ]]/{{ cookiecutter.code_directory }}
+    mount: /project/{{ cookiecutter.code_directory }}
+    local: {{ cookiecutter.code_directory }}
+  config:
+    remote: storage:$[[ flow.flow_id ]]/config
+    mount: /project/config
+    local: config
+    read_only: True
+  notebooks:
+    remote: storage:$[[ flow.flow_id ]]/notebooks
+    mount: /project/notebooks
+    local: notebooks
+  results:
+    remote: storage:$[[ flow.flow_id ]]/results
+    mount: /project/results
+    local: results
+  project:
+    remote: storage:$[[ flow.flow_id ]]
+    mount: /project
+    local: .
+
+images:
+  myimage:
+    ref: image:$[[ flow.flow_id ]]/develop
```

Comment:
A constant `develop` tag looks like a source of errors.
I can imagine a calculated tag which is produced from hashes of Dockerfile, requirements, apt-requirement files, etc.
The approach may require something like not-existing-yet `hash_files()` function (see github counterpart https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#hashfiles)
Adding the function to neuro-flow is not a hard task and can be an interesting exercise.

Reply:
ok I'm changing this to `ref: image:$[[ flow.flow_id ]]`. Note, we use implicit `:latest` here, should we fixate a specific version? 

cc @mariyadavydova